metrics.go

```diff
@@ -11,6 +11,7 @@ import (
 
 var metricsNameRegex = regexp.MustCompile(`\W+`)
 var metricsNameFixRegex = regexp.MustCompile(`^_+|_+$`)
+var VersionString = "dev"
```

Comment:
Maybe could you use https://pkg.go.dev/runtime/debug#ReadBuildInfo instead?

Reply:
I'll take a look!

Reply:
Actually, the version of the FrankenPHP module (and of all other installed modules) should already be exposed by Caddy: https://github.com/caddyserver/caddy/blob/v2.8.4/metrics.go#L15

Reply:
This is the string currently output from a production instance:

```
go_build_info{checksum="",path="github.com/dunglas/frankenphp/caddy",version="(devel)"} 1
```

It doesn't seem too useful :(

Reply:
Hum this looks like an issue in my release script 😅 (because of the replace). Isn't the version of lib (not the Caddy module) displayed?

Reply:
I'm not sure what you mean, but that is the only thing in the metrics.

Reply:
I mean we should have the version of `github.com/dunglas/frankenphp/` too

Reply:
There are no dependencies listed, just the top-level.